python/whylogs/experimental/core/udf_schema.py

```diff
@@ -32,20 +32,26 @@
 @dataclass
 class UdfSpec:
     """
-    Defines UDFs to apply to matching input columns. Tthe UDF is
+    Defines UDFs to apply to matching input columns. The UDF is
```

Comment:
this docstring should be updated to include matching by column type

Reply:
Yes, I will do the docstrings last once I'm sure the API is fully baked :)